src/Mod/PartDesign/Gui/TaskHoleParameters.cpp

```diff
@@ -426,24 +426,7 @@ void TaskHoleParameters::holeCutDepthChanged(double value)
 {
     PartDesign::Hole* pcHole = static_cast<PartDesign::Hole*>(vp->getObject());
 
-    if (ui->HoleCutCountersinkAngle->isEnabled()) {

```

Comment:
You cannot remove an existing feature. It is here because when you go deeper with the countersink tool, you get a larger diameter. So this feature exists on purpose.

Reply:
I tried to explain that in the forum entry, that is awaiting approval. The removed code was a one way route. It changes the diameter based on the difference of the depth. But manually changing the diameter doesn't change the hole depth. So you can get arbitrary depth values for a given diameter. This basically results in a meaningless depth parameter.
Maybe the original intention was to have a 1:1 relationship between depth and diameter. In that case the code shouldn't work on the difference, but on the absolute value and be implemented in both directions.